docs/guides/audio-tracks.md

```diff
@@ -1,69 +1,80 @@
+## Caveats
+- It is not possible to add audio tracks through HTML. They must be added [programmatically](#api).
+- Literal switching of audio tracks for playback is not handled by Video.js and must be handled by something else. Video.js only stores the track representation.
 
-- **Alternative**: alternative audio for the main video track
-- **Descriptions**: descriptions of what is happening in the video track
-- **Main**: the main audio track for this video
-- **Translation**: a translation of the main audio track
-- **Commentary**: commentary on the video, usually the director of the content talking about design choices
-
-## Missing Funtionality
-- It is currently impossible to add AudioTracks in a non-programtic way
-- Literal switching of AudioTracks for playback is not handled by video.js and must be handled by something else. video.js only stores the track representation
-
-## Adding to Video.js
-
-> Right now adding audio tracks in the HTML is unsupported. Audio Tracks must be added programatically.
-
-You must add audio tracks [programatically](#api) for the time being.
-
-## Attributes
-Audio Track propertites and settings
-
-### kind
-One of the five track types listed above. Kind defaults to empty string if no kind is included, or an invalid kind is used.
-
-### label
-The label for the track that will be show to the user, for example in a menu that list the different languages available for audio tracks.
+## Working with Audio Tracks
+### Add an Audio Track to the Player
+```js
+// Create a player.
+var player = videojs('my-player');
+
+// Create a track object.
+var track = new videojs.AudioTrack({
+  kind: 'translation',
+  label: 'Spanish',
+  language: 'es'
+});
 
-### language
-The two-letter code (valid BCP 47 language tag) for the language of the audio track, for example "en" for English. A list of language codes is [available here](languages.md#language-codes).
+// Add the track to the player's audio track list.
+player.audioTracks().addTrack(track);
+```
 
-### enabled
-If this track should be playing or not. In video.js we only allow one track to be enabled at a time. so if you enable more than one the last one to be enabled will end up being the only one.
+### Listen for a Track Becoming Enabled
+When a new track is enabled (other than the main track) an event is fired on the `AudioTrackList` called `"change"`. You can listen for that event and do something with it.
 
-## Interacting with Audio Tracks
-### Doing something when a track becomes enabled
-When a new track is enabled (other than the main track) an event is fired on the `AudioTrackList` called `change` you can listen to that event and do something with it.
-Here's an example:
 ```js
-// get the current players AudioTrackList object
-let tracks = player.audioTracks();
+// Get the current player's AudioTrackList object.
```

Comment:
I don't think we should showcase everything. I think the guides are meant to be a sort of tutorial or intro to working with them and not exhaustive API documentation.